pipeline.yml

```diff
@@ -274,7 +274,8 @@ stages:
               buildDefinition: 'pmc-sync'
               queueBuildForUserThatTriggeredBuild: true
               useSameSourceVersion: true
```

Comment:
I think this may need to be false as well?

Reply:
Sorry, I tested this in the pipelines repo and from github.

Reply:
Done